src/components/trips/TripSpeciesDetail.tsx

```diff
@@ -0,0 +1,24 @@
+import { useSpeciesDetail } from "@/hooks/useALASpeciesDetail"
+
+type SpeciesDetailProps = {
+  speciesId: string
+}
+
+export const TripSpeciesDetail = ({ speciesId }: SpeciesDetailProps) => {
+  const { data, error } = useSpeciesDetail(speciesId)
+
+  if (!data || error || data.error) {
+    return <></>
+  }
+
+  return (
+    <div>
```

Comment:
I would like to see this formatted as a card type component with:
- 1 image on the left, if there is one (the first image is fine)
- Display the image in an img tag
- One common name is enough
- include the species taxonomical name and the indigenous name in this component too